<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>KD6R309AX3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KD6R617AX3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KD6R926AX3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ボード数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBoardCount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>コンフィグファイル名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditConfigFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConfigFileName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>41</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ボード側ラインクロック</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBoardLineClock">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CISカメラ設定</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxResolution">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>600 DPI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300 DPI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150 DPI</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主走査解像度</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOuterTriggerMode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>外部ライントリガー同期</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LED制御</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLEDControl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>全点灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>片側（A)点灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>片側（B)点灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>消灯</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>点灯Duty比　全区間パルス数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>点灯Duty比　片側（A）パルス数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>点灯Duty比　片側（B)パルス数</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxLEDDutyFull">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>65535.000000000000000</double>
    </property>
    <property name="value">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxLEDDutyA">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>65535.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxLEDDutyB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>65535.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>照明分割点灯</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLEDDivide">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>分割無し</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2分割</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4分割</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8分割</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBoxGainMode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ゲイン</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>赤</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>緑</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>青</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxGainRed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxGainGreen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxGainBlue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxScanDirection">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ライン搬送方向</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxGamma">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ガンマ補正</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEvenOffsetY">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>偶数ピクセル　Yオフセット</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>奇数ピクセル　Yオフセット</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxOddOffsetY">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdjustWhite">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>白補正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdjustBlack">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>黒補正</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxModeAdjustWhite">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>白補正</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>750</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>750</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
